router/pump.go

```diff
@@ -91,15 +91,22 @@ func ignoreContainer(container *docker.Container) bool {
 
 	excludeLabel := cfg.GetEnvDefault("EXCLUDE_LABEL", "")
```

Comment:
For backwards compatibility, I didn't change the environment variable name. Although with this change, it would make more sense to be called `EXCLUDE_LABELS`

Reply:
Done